### PR TITLE
Measure load time without drift

### PR DESCRIPTION
**Pull request by joshaber**
Clock drift is my best current theory on why we've had a few reports of negative load time. Let's use methods that aren't based on the clock time.

### CODE REVIEW

File: app/src/ui/app.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class App extends React.Component<IAppProps, IAppState> {
       this.forceUpdate()
 
       requestIdleCallback(() => {
-        const now = Date.now()
+        const now = performance.now()
```
Comment (shiftkey) [inline app/src/ui/app.tsx]: Is there a reason why we're not using the same `process.hrtime` here? Is it not available in the renderer process?

Comment (shiftkey) [inline app/src/ui/app.tsx]: Oh, maybe it's the other way around and `window.performance` isn't available in the main process.

Anyway, a little shout out to this difference would be :gem:

**Comment by shiftkey** [comment]
:label: ❓

File: app/src/ui/app.tsx (hunk 1 above)
Comment (joshaber) [inline app/src/ui/app.tsx]: Yup, exactly. Added a note to the docs on `now`.

**Comment by joshaber** [comment]
🏷 🎤

**Pull request merged**